__tests__/app.test.js

```diff
@@ -276,6 +276,26 @@ describe("PATCH /api/articles/:article_id", () => {
         });
       });
   });
+  test("PATCH 200: when vote count is empty, or zero the article stays the same", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 0})
```

Comment:
I think Harrison meant that your test should have this inc_votes property be totally missing instead of 0.